osm-fudge/__main__.py

```diff
@@ -0,0 +1,27 @@
+import bk_tree
+import sys
+
+import levenshtein
+import jaccard
+
+distance_metrics = {
+	'levenshtein': levenshtein.find_levenshtein_distance,
+	'jaccard': jaccard.find_jaccard_similarity
+}
+
+with open(sys.argv[1], 'r') as file:
+	initial_word = file.readline()
+	metrics = sys.argv[2]
+	if metrics not in distance_metrics.keys():
+		sys.exit("Allowed metrics are {metrics}".format(metrics=list(distance_metrics.keys())))
+	options = {'n' : 2}
+	tree_obj = bk_tree.BKTree(initial_word, distance_metrics[metrics], options)
```

Comment:
I think it is more elegant if the bktree does not know about options for the metrics. By definition a metric is a function on two items (and then some metric properties); that's all the bktree cares about. It's probably better to decouple options for metrics and then only pass the resulting metric to the tree here.

Reply:
Got you, needs the options to be binded with metrics function

Reply:
Tried it with functools partial library

```
from functools import partial
```

Reply:
Perfect!